server/text_generation_server/utils/logits_process.py

```diff
@@ -20,17 +20,13 @@
     TypicalLogitsWarper,
 )
 
+from transformers.generation.logits_process import _calc_banned_ngram_tokens
+
```

Comment:
I believe ruff is erroring because `_calc_banned_ngram_tokens` is unused. Removing it should make the linter happy

```suggestion
```